docs/products/kafka/howto/create-topics-automatically.rst

```diff
@@ -1,31 +1,29 @@
 Creating Apache Kafka topics automatically
 ==========================================
 
-By default Aiven for Apache Kafka doesn't allow the automatic topic creation when a message is produced to a topic that does not yet exist. In such cases the common error message is the following::
+You can set Kafka to automatically create a topic when a message is produced to a topic that does not exist. 
+By default, in Aiven for Apache Kafka this features is turned off as safeguard against accidental topic creation and when a message is produced to a topic that does not yet exist. In such cases the common error message is the following::
 
     KafkaTimeoutError: Failed to update metadata after 60.0 secs.
 
 In such cases two options are available:
 
-#. create topics beforehand
-#. enable topic automatic creation
+#. Create topics beforehand
+#. Enable topic automatic creation
 
 While option 2 seems the easier solution it has drawbacks like the risk of creating new topics in case of typos or accepting all the :doc:`topic configuration default values <set-kafka-parameters>` set at service level such as partition count, replication factor, and retention time.
 
-Option 1 (create topics beforehand) is generally considered a good practice and suggested for production environments. However, for specific needs, you might want/need to enable automatic topic creation, which can be done by changing the ``auto_create_topics_enable`` parameter using the :doc:`Aiven CLI </docs/tools/cli>` or the `Aiven web console <https://console.aiven.io/>`_.
+Option 1 (create topics beforehand) is generally considered a good practice and suggested for production environments. However, for specific needs, you might want/need to enable automatic topic creation, which can be done by changing the ``auto_create_topics_enable`` parameter using the :doc:`Aiven CLI </docs/tools/cli>` or the `Aiven web console <https://console.aiven.io/>`_ by selecting the corresponding settings in the *Advanced configuration* section of the *Overview* page for your service.
```

Comment:
Would it sound better if instead "Option 1 (create topics beforehand) " we simply state "Creating topics beforehand is generally considered a  preferred practice and recommended". I'd also use "recommend" instead of "suggest" as a stronger alternative

Reply:
Agreed, changed!